.github/workflows/blocks-e2e.yml

```diff
@@ -0,0 +1,179 @@
+name: Run Blocks E2E Tests
+
+on:
+    pull_request:
+        paths-ignore:
+            - '**/changelog/**'
+            - '**/woo-ai/**'
+            - '**/woocommerce-admin/**'
+            - '**/woocommerce-beta-tester/**'
+            - '**/woocommerce-docs/**'
+            - '**/woocommerce/**'
```

Comment:
```suggestion
        paths:
            - 'plugins/woocommerce-blocks/**'
            - 'plugins/woocommerce/src/Blocks/**'
            - 'plugins/woocommerce/templates/**'
            - 'plugins/woocommerce/patterns/**'
```

Reply:
In this case I think it's better to target the the paths associated with Blocks. Generally speaking, `paths` is preferable unless you _specifically_ want a PR to run on every change _except_ the paths you defined.